### ISSUE TITLE
Could I use BatchNormalization in VGG and load the pre-trained weights

### ISSUE THREAD

**Issue by shijieS** [amdegroot/ssd.pytorch#126]
Hi,
  I have seen there an option for batch normalization for the vgg. Could I use it and load the pre-trained weights to train ssd?
